### PR TITLE
Combine Rates - Hybrid Data Source Only 

### PR DESCRIPTION
**Pull request by ailZhou**
### Description

This PR contains the hybrid measure calculation used in the Combined Rates section. Despite the careful planning withe administrative calculation, we still needed to do a couple of rewrites to accommodate `HybridMeasurePopulationIncluded` field we are pulling from the measures.. Its located in the **Definition of Population Included in the Measure** section.
 
Screenshot 2024-07-25 at 12 02 40 PM

**New Feature**
- The Combined Rates N/D/R table will add extra rows based on whether the data source has a hybrid selected.

**Side Issue Fix**: There was an issue with CHIP data source displaying in the data source information banner. It was caused by the naming update from CHIP to Separate CHIP

### Related ticket(s)

CMDCT-3652

---
### How to test

1) Sign into QMR, [stateuser1@test.com](mailto:stateuser1@test.com)
2) Select a Hybrid Measure to fill out, i.e. IMA-CH

Here are the possible state of the table based on what values were entered in.

State 1: No data source selected, no Measure-Eligible Population filled,  Rates filled
Screenshot 2024-07-25 at 12 06 15 PM

State 2: Medicaid data source selected, no Measure-Eligible Population filled, Rates filled 
Screenshot 2024-07-25 at 12 08 08 PM

State 3: Medicaid data source selected, Medicaid Measure-Eligible Population filled, Rates filled 
Screenshot 2024-07-25 at 12 08 56 PM

State 4: Both data source selected, Medicaid Measure-Eligible Population filled, Rates filled
Screenshot 2024-07-25 at 12 09 20 PM

State 5: Both data source selected, both Measure-Eligible Population filled, Rates filled
Screenshot 2024-07-25 at 12 09 44 PM

### Notes

N/A

---
### Pre-review checklist

- [x] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
- [x] I have performed a self-review of my code
- [x] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [x] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by BearHanded — APPROVED** [review]
My only concern is the casing for some of the labels and strings, but I think we're already far enough down the road in QMR that changing it would be a separate effort. 

Things like "measure-eligible population" "weighted rate" and "Casemanagementrecordreview"

File: services/app-api/handlers/rate/calculations/adminstrativeCalculation.ts [hunk 1]
```diff
@@ -45,17 +40,7 @@ export class AdminstrativeCalculation extends RateCalculation {
...
-      // This is good because we still want to calculate even if only one measure is complete.
-      const chipSourcesMatch = chipSources.every((chipSrc) =>
-        dataSrc.CHIP.includes(chipSrc)
-      );
-      const medicaidSourcesMatch = medicaidSources.every((medSrc) =>
-        dataSrc.Medicaid.includes(medSrc)
-      );
-      return chipSourcesMatch && medicaidSourcesMatch;
-    });
+    return super.check(arr);
```
Comment (rocio-desantiago) [inline services/app-api/handlers/rate/calculations/adminstrativeCalculation.ts:43]: what's 'super' here?

File: services/app-api/handlers/rate/calculations/hybridCalculation.test.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+const hybridData = [
+  {
+    DataSource: [DataSource.Hybrid],
+    DataSourceSelections: [],
+    PerformanceMeasure: {
+      rates: {
+        DFukSh: [
+          {
+            uid: "DFukSh.g91VU9",
```
Comment (rocio-desantiago) [inline services/app-api/handlers/rate/calculations/hybridCalculation.test.ts:14]: does test data need `uid` property? Noticed they're duplicated. Is that intentional?

**Review by ajaitasaini — APPROVED** [review]
tested in cloudfront 👍

File: services/app-api/handlers/rate/calculations/adminstrativeCalculation.ts (hunk 1 above)
Comment (BearHanded) [reply services/app-api/handlers/rate/calculations/adminstrativeCalculation.ts:43]: I'll shoot you a message on the side to chat about it, resolving the conversation so the PR can go through

**Review by angelaco11 — APPROVED** [review]
without full context, i went through the logic of this and it seems sound. also I tested and it all seems to be working like your screenshots

**Review by rocio-desantiago — APPROVED** [review]
have a question with test data. Thank you Brendan for showing me reference about `super`. I initially thought it was a variable but it's a keyword!

**Review by benmartin-coforma — APPROVED** [review]

**Pull request merged**